src/go/cmd/http-relay-server/broker.go

```diff
@@ -212,7 +221,7 @@ func (r *broker) PutRequestStream(id string, data []byte) bool {
 	return true
 }
 
-// SendResponse delivers the HttpResponse to the client handler that created the
+// SendResponse delivers the HttpResponse to the web-client handler that created the
```

Comment:
again, user-client?

Reply:
Found a few more places and fixed them too.